h/static/styles/partials-v2/_tooltip.scss

```diff
@@ -0,0 +1,74 @@
+@mixin tooltip-arrow($rotation) {
+  background: $grey-7;
+  border-bottom: 1px solid rgba(0,0,0,0.20);
+  border-right: 1px solid rgba(0,0,0,0.20);
+  content: "";
+  display: none;
+  height: 6px;
+  left: 0;
+  margin-left: auto;
+  margin-right: 7px;
+  position: absolute;
+  right: 0;
+  transform: rotate($rotation);
+  width: 6px;
+
+  .env-js-capable & {
+    display: block;
+  }
+
+  .env-js-timeout & {
+    display: none;
+  }
+}
+
+.tooltip {
+  @include font-small;
+
+  border-radius: 2px;
+  background-color: $grey-7;
+  color: white;
+  display: none;
+  font-style: normal;
+  padding-left: 5px;
+  padding-right: 5px;
+  padding-top: 4px;
+  padding-bottom: 4px;
+  position: fixed;
+  max-width: 220px;
+  z-index: $zindex-tooltip;
+
+  .env-js-capable & {
+    display: block;
+  }
+
+  .env-js-timeout & {
+    display: none;
+  }
+}
+
+// Arrow at the bottom of the tooltip pointing down at the target element.
+.tooltip:before {
+  @include tooltip-arrow(45deg);
+  content: "";
+  top: calc(100% - 5px);
+  right: calc(100% - 17px);
+}
+
+.tooltip-label {
+  // Make the label a positioned element so that it appears _above_ the
+  // tooltip's arrow, which partially overlaps the content of the tooltip.
+  position: relative;
+}
+
+@include touch-input {
+  .form-input__label .tooltip {
```

Comment:
Components are meant to be reusable and isolated from their surrounding context. This means that we shouldn't style the tooltip in a specific context (eg. when used on a form input) within the file that defines the styles for the component. There are different ways to address the problem:

1. Try to restructure the styling / component in such a way that you don't need the contextual styles. Here for example, the '50px' is actually used because the styling is tied to the width of the "ORCID" label. We should try to find a way of setting up the tooltip so that the margins are independent of this
2. If you do need multiple variations of a component, eg. Danger / Cancel / Normal buttons, you can use `component--modifier` styles.
3. If you really do need some context-specific adjustment, you can put the element in a wrapper `<span>` or `<div>`. The search bar in the navbar for example is wrapped in a `nav-bar__search` element which handles positioning of the search bar within the navbar, so that the search input can be reused elsewhere.